tensorflow_cloud/__init__.py

```diff
@@ -22,3 +22,4 @@
 from .machine_config import COMMON_MACHINE_CONFIGS
 from .machine_config import MachineConfig
 from .run import run
+from .remote import remote
```

Comment:
Unit tests are failing. Try from .run import remote

Reply:
Good catch! Thank you.